internal/integration/integration_test.go

```diff
@@ -236,7 +236,7 @@ func TestIntegration(t *testing.T) {
 
 		talosConfig := createTalosConfig(ctx, t, c, namespaceName, bootstrapv1alpha3.TalosConfigSpec{
 			GenerateType: talosmachine.TypeControlPlane.String(),
-			TalosVersion: TalosVersion,
+			TalosVersion: "v0.13",
```

Comment:
this test is using legacy secret data (from old CABPT) which doesn't support cluster identity secrets required for cluster discovery (which is default in 0.14)